api/HTMLUnknownElement.json

```diff
@@ -5,10 +5,10 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLUnknownElement",
         "support": {
           "chrome": {
-            "version_added": true
+            "version_added": "15"
```

Comment:
I don't have Chrome 14 to test with, but I tested Safari 4, which is a WebKit version soon before this, and can see that `document.createElement('bla')` returns a `HTMLElement`, not `HTMLUnknownElement`. So this seems correct. I had to go forward to Safari 6 to see it supported, while I expected 5, but nevertheless I trust these results based on how http://mdn-bcd-collector.appspot.com/tests/api/HTMLUnknownElement works.